cmd/gossamer/config.go

```diff
@@ -553,6 +555,13 @@ func setDotRPCConfig(ctx *cli.Context, tomlCfg ctoml.RPCConfig, cfg *dot.RPCConf
 		cfg.Enabled = false
 	}
 
+	// check --rpc-external flag and update node configuration
+	if external := ctx.GlobalBool(RPCExternalFlag.Name); external {
+		cfg.External = true
```

Comment:
could you update this to also set `cfg.Enabled = true`? ie. if someone specifies `--rpc-external` but not `--rpc` it should still turn on the server